### ISSUE TITLE
Scrapland Remastered hangs before getting to the main menu

### ISSUE THREAD

**Issue by WinterSnowfall** [AlpyneDreams/d8vk#36]
![SR](https://user-images.githubusercontent.com/6306593/197347031-8f0724cb-887d-44d1-b8f2-e75f1e8dd794.png)

>warn:  D3D8DeviceEx::UpdateTexture STUB (IDirect3DBaseTexture8* pSourceTexture, IDirect3DBaseTexture8* pDestinationTexture) -> HRESULT

gets printed repeatedly in the logs and the rendering hangs.

### Software information
Scrapland Remastered

### System information
- GPU: Nvidia GTX 1080
- Driver: 515.76
- Wine version: 7.19 Staging
- DXVK version: 7cd4c8b

### Apitrace file(s)
Can capture one if needed, but it's probably just an effect of the stub above.

### Log files
- d3d8.log: [Scrap_d3d8.log](https://github.com/AlpyneDreams/d8vk/files/9844683/Scrap_d3d8.log)
- d3d9.log: [Scrap_d3d9.log](https://github.com/AlpyneDreams/d8vk/files/9844684/Scrap_d3d9.log)

**Comment by WinterSnowfall** [comment]
![Scrapland](https://user-images.githubusercontent.com/6306593/197565170-a2ed0476-8a71-4e54-ae1a-4102199c093c.png)

Alright, ever since e0e58bf the game works just fine :tada:. That's the main menu in the screenshot, but I played the game for a bit and it seems fine.

The logs are still quite noisy, so I'll leave it up to you if you want to investigate any of the warnings any further: [Scrap_d3d9.log](https://github.com/AlpyneDreams/d8vk/files/9852932/Scrap_d3d9.log)

**Comment by AlpyneDreams** [comment]
- `warn:  ConvertFormat: Unknown format encountered: 65` fixed by 4fb7acc64e29b7e82436817ac04d8f02a09ea255
- `D3D9DeviceEx::QueryInterface: Unknown interface query` fixed by d1cb703e0013fa6dceb4bc4b0c08695fe9418944
- `warn:  D3D9DeviceEx::SetRenderState: Unhandled render state 47` fixed by 5fa579db44e2b512db699cc6578ef5fddc1ce01e
- Render state 26 is dithering (#52)

So that just leaves these:
```
warn:  ConvertFormat: Unknown format encountered: 909200416
warn:  ConvertFormat: Unknown format encountered: D3D9Format::AL16
warn:  ConvertFormat: Unknown format encountered: D3D9Format::R16
warn:  ConvertFormat: Unknown format encountered: 909201985
```

- 909200416 is `" L16"`
- 909201985 is `"AR16"`

I've opened a new issue #68 for this. Any other issues with this game should be a separate issue.
